cis-benchmarks/README.md

```diff
@@ -0,0 +1,29 @@
+# CIS Benchmarks
+
+This plugin utilizes the [kube-bench][kubebench] implementation of the [CIS security benchmarks][cis]. It is technically two plugins; one to run the checks on the master nodes and another to run the checks on the worker nodes.
+
+## Usage
+
+To run this plugin, run the following command:
+
+```
+sonobuoy run
+--plugin https://raw.githubusercontent.com/johnSchnake/sonobuoy-plugins/cis-benchmarks/cis-benchmarks/kube-bench-plugin.yaml --plugin https://raw.githubusercontent.com/johnSchnake/sonobuoy-plugins/cis-benchmarks/cis-benchmarks/kube-bench-master-plugin.yaml 
```

Comment:
Now that this will be in the `vmware-tanzu` repo, these links should be updated.